### PR TITLE
Inconsistent scoped alias

### PR DESCRIPTION
**Pull request by dpassen**
Addresses #1780 

Please answer the following questions and leave the below in as part of your PR.

- [X] I have read the [developer documentation](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md).

- [X] This PR corresponds to an [issue with a clear problem statement](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [X] This PR contains a [test](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#tests) to prevent against future regressions

- [X] I have updated the [CHANGELOG.md](https://github.com/clj-kondo/clj-kondo/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

**Comment by borkdude** [comment]
Looks good to me so far. As this is a Draft, let me know when it's ready to go.

**Comment by dpassen** [comment]
I’ll need to update the change log and squash, if that’s preferred.

File: src/clj_kondo/impl/analyzer/namespace.clj [hunk 1]
```diff
@@ -83,7 +83,9 @@
 
 (defn lint-alias-consistency [ctx ns-name alias]
   (let [config (:config ctx)]
-    (when-let [expected-alias (get-in config [:linters :consistent-alias :aliases ns-name])]
+    (when-let [expected-alias (or
+                               (get-in config [:linters :consistent-alias :aliases ns-name])
```
Comment (borkdude) [inline src/clj_kondo/impl/analyzer/namespace.clj:87]: One minor thing, for performance reasons you could get `(get-in config [:linters :consistent-alias :aliases])` only once and then re-use that result.

**Comment by borkdude** [comment]
@dpassen No need to force-push, your commits will be squashed when merged.

File: src/clj_kondo/impl/analyzer/namespace.clj (hunk 1 above)
Comment (dpassen) [reply src/clj_kondo/impl/analyzer/namespace.clj:87]: Yes of course

**Comment by borkdude** [comment]
Thanks!

**Pull request merged**